core/primitives/src/types.rs

```diff
@@ -40,17 +40,11 @@ pub type BlockIndex = u64;
 pub type ShardId = u64;
 /// Balance is type for storing amounts of tokens.
 pub type Balance = u128;
-/// Identifier for receipts, used to refer in callbacks.
-pub type ReceiptId = Vec<u8>;
-/// Identifier for callbacks, used to store storage and refer in receipts.
-pub type CallbackId = Vec<u8>;
+/// Gas is a type for storing amount of gas.
+pub type Gas = u64;
```

Comment:
we called in GasUsed in our branch, also can GasAmount? just Gas seems a bit confusing to me

Reply:
@nearmax  also uses `Gas` in his branch. We can switch to `GasAmount`, but seems `Gas` is not used anywhere else, so it can be a type for `gas`?

Reply:
`Gas` seems fine to me, since we don't have different underlying types for `GasUsed`, `GasBurnt` etc (for the contrast for storage we had `StorageUsage` and `StorageUsageChange` because one is `u64` and another is `i64`). It is not part of the protobuf or the protocol, so we can always change it later if we find it confusing or our code gets more complex.